README.md

```diff
@@ -3,10 +3,10 @@
 The Kani Rust Verifier is a bit-precise model checker for Rust.
 
 Kani verifies:
- * memory safety -- even in unsafe Rust code
- * user-specified assertions
- * absence of panics
- * checks for some classes of undefined behavior
+ * Memory Safety -- Even in unsafe Rust code.
+ * User specified Assertions.
+ * Absence of Panics.
+ * Checks for some classes of Undefined Behavior.
```

Comment:
Capitalizing first letters seems fine, but the rest of these appear to be random capitalization. (People frequently have a tendency to capitalize "meaningful words" but this isn't usually an improvement and is an impulse it's best to try to get over. I completely get where you're coming from with e.g. capitalizing UB, but it's better just writing the words normally. :P)

Reply:
haha, sure. I'll undo those.